otterdog/eclipse-basyx.jsonnet

```diff
@@ -306,5 +306,17 @@ orgs.newOrg('eclipse-basyx') {
         enabled: false,
       },
     },
+    orgs.newRepo('basyx-wiki') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      default_branch: "master",
+      delete_branch_on_merge: false,
+      secret_scanning: "disabled",
```

Comment:
new repos should always created with secret scanning / push protection enabled
you get it for free and avoid stupid mistakes